products/rtpengine.md

```diff
@@ -0,0 +1,48 @@
+---
+title: rtpengine 
+category: server-app
+permalink: /rtpengine
+versionCommand: rtpengine -v
+releaseDateColumn: true
+
+identifiers:
+-   repology: rtpengine
+auto:
+  methods:
+    - git: https://github.com/sipwise/rtpengine.git
+      regex: ^mr(?P<major>\d+)_(?P<minor>\d+)_(?P<patch>\d{1,3})_?(?P<tiny>\d+)?$
```

Comment:
```suggestion
      regex: '^mr(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\.(?P<tiny>\d+))?$'
```

Add a look at https://github.com/sipwise/rtpengine/tags, the separator is `.`. Also simplified the regex a bit.

Reply:
Checked locally, works as expected.